released/packages/coq-dblib/coq-dblib.8.6.0/opam

```diff
@@ -0,0 +1,14 @@
+opam-version: "1.2"
+maintainer: "matej.kosik@inria.fr"
```

Comment:
If you have created this package
(i.e. `descr`, `opam`, `url` files)
and especially:
- if you have tested this OPAM package
- if you plan to keep it installable with the indicated Coq version

then I think it makes sense to make sure that people can contact **you** instead of me.

What do you think?